schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/DefaultActivityService.java

```diff
@@ -134,33 +109,23 @@ private void validateRepeatDetails(RepeatDetails repeatDetails) throws FieldValu
 
     @Nonnull
     @Override
-    public Key addActivityType(Organization organization, final ActivityType activityType) throws EntityNotFoundException, UniqueConstraintException, FieldValueException {
-        final String name = StringUtils.trimToNull(activityType.getName());
-        if (name == null) {
-            throw new FieldValueException("ActivityType.name");
-        }
```

Comment:
Here I'm a little bit in doubt.  The fact that `activytType.name` must not be empty is related to the integrity of the model, so that might be something we need to check both on the controller (`XXXEndpoint`) and the model (`XXXDao`).  Handle it nicely on the Endpoint and protective on the DAO, eg:
```java
Preconditions.checkArgument(StringUtils.isNonBlank(name))
```

Just a thought...  The DAO's responsible for keeping the model's integrity IMO